compatibility-verifier/sample-test-suite/config/dataGenerator.py

```diff
@@ -0,0 +1,90 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import random
+import csv
+import string
+
+class DataGenerator:
```

Comment:
I think this is used to generate featuretest3? Is it one time (I hope) or on each run (in which case I have more questions)? Either way, please add a comment on this file. Thanks.

Reply:
Oh it's a one time generation, putting it here for future reference and. Will add comment.

Reply:
Can you add a comment with the class that this is a one-time usage to generate data? thanks